src/common/schemas/objects/instructions.json

```diff
@@ -14,7 +14,7 @@
       "$ref": "value"
     },
     "maxFee": {
-      "description": "The maximum fee to pay for the transaction. See [Transaction Fees](#transaction-fees) for more information.",
+      "description": "The maximum fee to pay for this transaction. If this exceeds the `maxFeeXRP` set in the RippleAPI constructor, `maxFeeXRP` will be used instead. See [Transaction Fees](#transaction-fees) for more information.",
```

Comment:
What do you think about removing `maxFee`?

Reply:
I think there's a use case for some transactions being higher priority than others. For example, if an existing user has set `maxFee` to `0.5` to ensure more economical transactions, I don't want to break their integration.

Reply:
After a quick chat, we've decided to deprecate `maxFee`. It will be removed entirely in a future release of ripple-lib (perhaps 2.0.0).